src/frontend/data/stores/useVideo/index.tsx

```diff
@@ -27,10 +27,25 @@ export const useVideo = create<VideoState>((set, get) => ({
         videosToAdd,
       ) as VideoStateResource,
     ),
-  addResource: (video: Video) =>
-    set(
+  addResource: (video: Video) => {
+    // TODO: Remove this hack
+    // This is a hack to keep existing jitsi infos in updated videos.
+    // Should be removed when we have a proper way to send those infos
+    // through websocket for participants in discussion.
+    const lastVideoState = get()[modelName.VIDEOS][video.id];
+    if (lastVideoState?.live_info?.jitsi) {
+      video = {
+        ...video,
+        live_info: {
+          ...video.live_info,
+          jitsi: lastVideoState.live_info.jitsi,
+        },
+      };
+    }
```

Comment:
Wouldn't it be easier to expose in the backend an enpoint to fetch jitsi infos for a video and guard this endpoint to let only people accepted on stage to be able to fetch those infos ?

Reply:
For now we don't have a reliable way to do that.
But it's planned with the anonymous id.

Reply:
It's not directly linked to the anonynmous_id feature but how we are refactoring the live_session usage. Neverless, this temporary hack is the best solution found while we don't have the websocket targeting only people on stage or asking to join the discussion.

Reply:
> we should prompt displayed name form for anonymous users who want to use the 'on stage' feature, isn't it ?

This is not the purpose of this PR. You are true about what we have to do but this is out of the scope of this PR